packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1861,6 +1864,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this.garbageCollector.collectGarbage(options);
     }
 
+    /**
+     * Called when a new reference is added to another Fluid object. This is required so that garbage collection can
+     * identify all references added in the system.
+     * @param id - The id of the node that added the reference.
+     * @param referencedHandle - The handle of the Fluid object that is referenced.
+     */
+    public referenceAdded(id: string, referencedHandle: IFluidHandle) {
```

Comment:
Anything we can do with name or comments to make it clearer what "reference" is? I'm not sure people having less insights into all the details of GC / handle management would immediately get semantics and expectations

Reply:
How about `addedOutboundReference` or `addedOutboundRoute`?

Reply:
Maybe addedGCOutboundRoute? To add GC context

Reply:
Changed it to `addedGCOutboundReference`.